e2e/tests/update_params_test.go

```diff
@@ -172,8 +191,56 @@ func (s *suite) TheModuleParamShouldBeUpdated(moduleName, paramName string) {
 	s.assertExpectedModuleParamsUpdated(moduleName)
 }
 
+// TheModuleParamShouldBeSetToItsDefaultValue asserts that the given param for the
+// given module has been set to its default value.
+func (s *suite) TheModuleParamShouldBeSetToItsDefaultValue(moduleName string, paramName string) {
+	// TODO_HACK: So long as no other modules are expected to have been changed by this scenario,
+	// it is more than sufficient (and less code) to re-use the existing step which asserts that
+	// all modules have their params set to their respective defaults.
+	_ = paramName
+	s.AllModuleParamsShouldBeSetToTheirDefaultValues(moduleName)
+}
+
+// ensureAccountForKeyName ensures that an account exists for the given key name in the keychain.
+func (s *suite) ensureAccountForKeyName(keyName string) {
+	s.Helper()
+
+	// Get the address of the key.
+	addr := s.getKeyAddress(keyName)
+
+	// Fund the account with minimal tokens to ensure it is on-chain.
+	s.ensureOnChainAccount(addr)
+}
+
+// ensureOnChainAccount sends a minimal amount of upokt tokens to the given address
+// to ensure it has an on-chain account.
```

Comment:
```suggestion
// ensureOnChainAccount sends a minimal amount of upokt tokens to the given address
// to ensure it has an on-chain account. This means that the public key is retrievable when
// queried for by address.
```

Reply:
> This means that the public key is retrievable when queried for by address.

I don't believe that this is accurate. Funding the address ensures that it has a balance in the bank module; however, it doesn't have any effect on the auth module AFAIK, which is where the public key is stored. My understanding is that a tx would have to be submitted which was signed by that key in order to get the public key on-chain. While funding the address is a prerequisite for this, they are distinct steps.

Reply:
I did a bit of renaming/refactoring to improve readability:

![image](https://github.com/pokt-network/poktroll/assets/600733/b9e88ea9-28f5-426c-9ced-2ccae79209fc)

![image](https://github.com/pokt-network/poktroll/assets/600733/71243d68-5a7d-473a-8588-d8a119ef8fee)